### DISCUSSION TITLE
Option to show list of only filenames in search results

### DISCUSSION THREAD

**Discussion by OnkarRuikar** [community/community#16537] [Other Feature Feedback, Questions, & Ideas]
It would be nice if we can get list of only file names with number of matches. 
The list can be easily shared in issue/bug discussions or over emails. For example, https://github.com/mdn/content/issues/15872#issue-1231382488
The  list can be used to easily assign tasks, like update documents containing certain outdated terms/methods.

Something like this:
![file list in results](https://i.imgur.com/bwOO7wO.png)

**Comment by dsumac** [comment]
We have the same need in our organization.

Examples:
- get all files / projects concerned by a dependency
